src/windows/views/transitions_treeview.py

```diff
@@ -59,7 +59,7 @@ def startDrag(self, event):
 
         if not current.isValid():
             # We can't find anything to drag
-            log.warning("No draggable items found in model!")
+            log.warning("No draggable items found in TransitionsTreeView model!")
```

Comment:
Here too, I'm afraid. :wink:

Reply:
(In case you needed evidence that cut-and-paste code duplication is a major concern, here it is. Most of these would be logging with the _wrong_ class name, if the original had included the name of the class in the message. Nobody ever fixes those things up after copying the code.)